studio/config/docker/Dockerfile

```diff
@@ -45,4 +45,11 @@ COPY studio /app/studio
 COPY sample_data /app/sample_data
 COPY main.py alembic.ini ./
 
+# Copy the cloud-startup.sh script
+COPY studio/app/cloud-startup.sh ./
+RUN chmod +x cloud-startup.sh
+
 EXPOSE 8000
+
+# Set the entrypoint to run cloud-startup.sh
+ENTRYPOINT ["./cloud-startup.sh"]
```

Comment:
It is considered to be easier to maintain if you extract the files from the Dockerfile.
- Development environment ... in docker-compose.yaml (not required)
-  ECS environment ... in taskdef